package.json

```diff
@@ -4,23 +4,24 @@
   "version": "0.0.0",
   "scripts": {
     "build": "lerna run build --stream",
-    "build:demo": "build-storybook -s packages/bedrock/dist -o demo",
+    "build:demo": "storybook build -s packages/bedrock/dist -o demo",
     "format": "prettier-package-json --write && lerna run format --stream",
     "postinstall": "husky install && lerna bootstrap",
     "lint": "eslint scripts/*.js && lerna run lint --stream",
     "prepare": "yarn build",
-    "start": "start-storybook -s packages/bedrock/dist -p 6006",
+    "start": "storybook dev -s packages/bedrock/dist -p 6006",
```

Comment:
while we're at it, can we add the [no-version-updates](https://github.com/zendeskgarden/react-components/blob/4d7f950f6cfaf99a87fbb4bd66eb1ee3818191bf/package.json#L22) flag (if it still exists)?

Reply:
It no longer exists according to the [new CLI docs](https://storybook.js.org/docs/react/api/cli-options) unfortunately.

Reply:
According to `yarn storybook dev --help` it is still there.